flask_jwt/__init__.py

```diff
@@ -50,7 +50,10 @@ def _default_jwt_payload_handler(identity):
     iat = datetime.utcnow()
     exp = iat + current_app.config.get('JWT_EXPIRATION_DELTA')
     nbf = iat + current_app.config.get('JWT_NOT_BEFORE_DELTA')
-    identity = getattr(identity, 'id') or identity['id']
+    if hasattr(identity, 'id'):
+        identity = getattr(identity, 'id')
+    else:
+        identity = identity['id']
```

Comment:
Why not `identity = getattr(identity, 'id', None) or identity['id']` as suggested in https://github.com/mattupstate/flask-jwt/issues/115#issue-207753121 ?